src/tooltip/test/tooltip.spec.js

```diff
@@ -33,7 +33,7 @@ describe('tooltip', function() {
   }));
 
   it('should open on mouseenter', inject(function() {
-    elm.trigger( 'mouseenter' );
```

Comment:
I would undo those changes - they can be tackled in a separate PR, but they pollute the feature.